lang/eiffelstudio/Portfile

```
@@ -17,29 +17,23 @@ long_description  EiffelStudio is a development environment for the \
                   The user interface rests on a number of specific UI paradigms, \
                   in particular "pick-and-drop" for effective browsing.
 homepage          http://www.eiffel.com
-master_sites      https://ftp.eiffel.com/pub/beta/${major_version}/:source \
-                  https://github.com/roederja/eiffel-launcher/releases/download/20201029/:launcher
+master_sites      https://www.eiffel.com/cdn/EiffelStudio/${major_version}/${minor_version}/pp/:source
```

Minor nit, but this can use the `${homepage}` variable:

```suggestion
master_sites      ${homepage}/cdn/EiffelStudio/${major_version}/${minor_version}/pp/:source
```